src/families/ethereum/modules/erc1155.ts

```diff
@@ -14,6 +14,9 @@ import { prepareTransaction } from "./erc721";
 const notOwnedNft = createCustomErrorClass("NotOwnedNft");
 const notEnoughNftOwned = createCustomErrorClass("NotEnoughNftOwned");
 const notTokenIdsProvided = createCustomErrorClass("NotTokenIdsProvided");
+const quantityNeedsToBePositive = createCustomErrorClass(
```

Comment:
errors need to start with uppercase (indeed it's also a mistake for all other vars)

Reply:
to be taken into consideration later